src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -106,11 +106,18 @@ and, optionally,
  - `q` [`PhasePartition`](@ref). Without this argument, the results are for dry air.
 """
 linearized_air_pressure(ρ::FT, e_tot::FT, e_pot::FT, q::PhasePartition{FT}=q_pt_0(FT)) where {FT<:Real} =
-  ρ*FT(R_d)*FT(T_0) + FT(R_d)/FT(cv_d)*(ρ*e_tot - ρ*e_pot - (ρ*q.tot - ρ*q.liq)*FT(e_int_v0) + ρ*q.ice*(FT(e_int_i0) + FT(e_int_v0)))
+  linearized_air_pressure_given_densities(ρ, ρ*e_tot, ρ*e_pot, ρ*q.tot, ρ*q.liq, ρ*q.ice)
 
 linearized_air_pressure(e_kin::FT, e_pot::FT, ts::ThermodynamicState{FT}) where {FT<:Real} =
   linearized_air_pressure(air_density(ts), total_energy(e_kin, e_pot, ts), e_pot, PhasePartition(ts))
 
+function linearized_air_pressure_given_densities(ρ::FT, ρe_tot::FT, ρe_pot::FT,
```

Comment:
I'd move this to `linear`, and make it one function. We do not usually have `..._given_densities` because given densities is the default.

Reply:
Done